prow/plugins/releasenote/releasenote.go

```diff
@@ -81,11 +84,12 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 		return nil
 	}
 
-	// Only allow authors and assignees to add labels.
-	isAssignee := ic.Issue.IsAssignee(ic.Comment.User.Login)
+	// Only allow authors and org members to add labels.
+	isMember, _ := gc.IsMember(trustedOrg, ic.Comment.User.Login)
```

Comment:
True, but by logging the error, we make it easier to debug in the event that it returns false for avalid contributor (i.e. we know if it returned false due to `404`, `302` or some other status code)